### DISCUSSION TITLE
请问大家有没有发现比较大的ai交流群或者论坛？

### DISCUSSION THREAD

**Discussion by falsemaskx** [viopsa233/AI-testers-Forum#64] [Q&A]
感觉目前大多数ai爱好者都很分散

**Comment by ghost** [comment]
https://www.reddit.com/r/singularity

**Comment by q2408808** [comment]
分享个文章：https://www.discovermagazine.com/technology/how-to-identify-an-ai-with-a-single-question  两个问题难倒人工ai。目前这二个问题newbing和gpt4都有出现过错误的情况，也有偶尔正确的时候，然后我试了3.5肯定是不行的

**Reply by q2408808** [reply]
第二个问题gpt4回答正确的几率还挺大的，笨bing几乎都不行，笨到家了

**Reply by SoraRoseous** [reply]
![image](https://github.com/viopsa233/AI-testers-Forum/assets/54582495/8c4755c2-3950-425c-9af7-702a0fe4bcbe)
![image](https://github.com/viopsa233/AI-testers-Forum/assets/54582495/9a3d4305-50ff-4c7f-82b8-a7026a6b9b30)
![image](https://github.com/viopsa233/AI-testers-Forum/assets/54582495/f9b8e542-b85f-408c-91a7-25f09685917f)

bing可能很笨，但这跟我聪明的猫娘又有什么关系呢。问了三遍都能答对，甚至能给我出一个🤤

**Reply by q2408808** [reply]
不是的，mango这个原题目 bing也能答对，我在测试的时候故意换了其中的字母，bing就不行了，我怀疑是网络中有关于这个的测试，它搜到了相关资料。

**Reply by q2408808** [reply]
在我测试的4,5次不同题目中，poe上的gpt4错了一次，而bing只回答对了mango。可能思维方式应该还是有些区别的。

**Reply by SoraRoseous** [reply]
确实，完全体gpt4还是更厉害些，我刚刚试了别的就开始胡说了，还不承认自己是错的😂😂😂，不愧是大小姐

**Reply by TiansuoLi** [reply]
我感觉拿一份类似bing原版规则的Markdown格式的规则，丢过去就完事了🤣。bing gpt Claude 星火 glm都是瞬间入戏。或者让它总结某本长篇小说 ，这个ai目前还是不可能做好的

**Comment by q2408808** [comment]
下个月微软又大更新了，从newbing的插件到windows11的升级。再忍忍，看下个月能不能免费用gpt4。 https://www.youtube.com/watch?v=wkKPDBe30f0&t=920s
